weave-js/src/components/PagePanelComponents/Home/Browse3/compare/ComparePage.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Dispatch to call or object specific comparison based on query parameters.
+ */
+
+import React from 'react';
+import {useHistory} from 'react-router-dom';
+
+import {Empty} from '../pages/common/Empty';
+import {
+  getParamArray,
+  queryGetBoolean,
+  queryGetDict,
+  queryGetString,
+} from '../urlQueryUtil';
+import {ComparePageCalls} from './ComparePageCalls';
+import {ComparePageObjects} from './ComparePageObjects';
+
+type ComparePageProps = {
+  entity: string;
+  project: string;
+};
+
+export const ComparePage = ({entity, project}: ComparePageProps) => {
+  const history = useHistory();
+  const d = queryGetDict(history);
+  const mode =
+    queryGetString(history, 'mode') === 'unified' ? 'unified' : 'parallel';
+  const baselineEnabled = queryGetBoolean(history, 'baseline', false);
+  const onlyChanged = queryGetBoolean(history, 'changed', false);
+  const callIds = getParamArray(d, 'call');
+
+  if (callIds.length) {
```

Comment:
I feel like we might want to limit the number to a reasonable set: 
![Screenshot 2024-11-19 at 15 23 18](https://github.com/user-attachments/assets/a4b54e6d-2e2b-402a-9025-4a9d6d10f9f9)

Reply:
this pretty much halts to page

Reply:
I'm going to limit it to 6 object columns in the grid and show a message. I think that will work for most cases while keeping the entire set in the shopping cart so you can select the items for a pairwise comparison.